packages/strapi/lib/middlewares/cron/index.js

```diff
@@ -37,6 +37,28 @@ module.exports = strapi => {
             taskValue.task
           );
         });
+
+        _.forEach(_.keys(strapi.plugins), pluginName => {
+          const pluginCron = strapi.plugins[pluginName].config.functions.cron;
```

Comment:
Hi @boazpoolman ! 

I think you need to test each part, because I've test on my side and if a plugin doesn't have a `config` folder this throw an error : 
```
TypeError: Cannot read property 'cron' of undefined
``` 

I've add this verification and it's working (but need to be improve I think) : 
```
if (strapi.plugins[pluginName].config &&
            strapi.plugins[pluginName].config.functions &&
            strapi.plugins[pluginName].config.functions.cron) {
            const pluginCron = strapi.plugins[pluginName].config.functions.cron;
            ....
``` 

And just for fun, I think we are connected, because I was needed of this feature and I've ask to @derrickmehaffy , then, 30 minutes later you submit your PR 😆

Reply:
Haha I think we are indeed connected 😛
Thanks for reviewing my PR!
I've updated it accordingly, slightly different than your solution.
Should work now!